ClassPaulita/GuessTheCardGame.cs

```diff
@@ -26,39 +160,60 @@ private bool UserChoseToQuit(char userChoice)
             return userChoice == 'q' || userChoice == 'Q';
         }
 
-        private bool IsUserChoiceValid(char userChoice)
-        {                      
+        private char GetUserGuess(Card previousCard)
+        {
+            Console.Write($"\n\n\tIs the next card greater than {previousCard}(y or n)? ");
+            char userGuess = Convert.ToChar(Console.ReadLine());
 
-            if (UserChoseToPlay(userChoice) || UserchoseToSeeInstructions(userChoice) || UserChosetoSeeBoard(userChoice) || UserChoseToQuit(userChoice))
+            while (!IsUserGuessValid(userGuess))
             {
-                return true;
+                Console.WriteLine("\n\tYour answer is not valid. Please try again!");
+                Console.Write($"\n\n\tIs the next card higher or lower than {previousCard}(y or n)? ");
+                userGuess = Convert.ToChar(Console.ReadLine());
```

Comment:
Same issue as above

Reply:
This is no longer in the correct order so it's my bad for saying "Same as the one above".
The issue here is the Convert.ToChar(Console.ReadLine());

Reply:
No problem. I was able to fix it. Now I am taking a string and checking its length before checking the first character.